src/styles/layout/grid-nested-annotate.scss

```diff
@@ -21,16 +21,8 @@ $grid-nested-annotate-row-background-color: #ffffff;
     }
 
     [class^="govuk-grid-column"] > p {
-      @include govuk-font($size: 24);
-      @include govuk-responsive-padding(6);
-
-      display: block;
-      margin: 0;
-
       color: govuk-colour("white");
       background-color: govuk-colour("blue");
-
-      text-align: center;
```

Comment:
Confirming that these styles were duplicated in [`grid-annotate.scss`](https://github.com/alphagov/govuk-design-system/blob/5a5db22c55e669bb5e0949c4105b489a37a9bfc8/src/styles/layout/grid-annotate.scss) 👍